REFERENCE.md

```diff
@@ -156,6 +157,17 @@ skips all mounting.  Note that migrating between for instance *autofs* and then
 
 Default value: `'autofs'`
 
+##### <a name="-cvmfs--config_repo"></a>`config_repo`
+
+Data type: `Optional[Stdlib::Fqdn]`
+
+When using the `mount_method` as `mount` it may be nescessary to specify a CvmFS located configuration_repository.
```

Comment:
Just a small nit: I think everywhere else we write `cmvfs` or `CVMFS`, i.e. either all caps or all lower case.